cli/src/main/java/io/kestra/cli/commands/tenants/TenantsCommand.java

```diff
@@ -0,0 +1,29 @@
+package io.kestra.cli.commands.tenants;
+
+import io.kestra.cli.AbstractCommand;
+import io.kestra.cli.App;
+import io.micronaut.configuration.picocli.PicocliRunner;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "tenants-oss",
```

Comment:
```suggestion
    name = "tenants",
```

You have to make this command do something else in EE, not provide two different commands